CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+Added feature: selectively ignore some of the measures (iotagent-json#416)
```

Comment:
```suggestion
Add: explicitAttrs flag (in configuration and also group/device provisioning) to progress only the measures correspoing to attributes declared in attributes array (iotagent-json#416)
```

I think that way the description is more precise. What do you think?

Reply:
Fixed in 339d67a